runner/llamarunner/runner.go

```diff
@@ -622,8 +617,6 @@ func (s *Server) completion(w http.ResponseWriter, r *http.Request) {
 				return
 			}
 
-			seq.crossAttention = s.image.NeedCrossAttention(seq.cache.Inputs...)
```

Comment:
There are a few other references to `crossAttention` in this file - they can all be removed.

Reply:
Still a couple more in `processBatch` that are dead code now.